lib/security-release/security-release.js

```diff
@@ -107,3 +121,66 @@ export function formatDateToYYYYMMDD(date) {
   // Concatenate year, month, and day with slashes
   return `${year}/${month}/${day}`;
 }
+
+export async function createIssue(title, content, repository, { cli, req }) {
+  const data = await req.createIssue(title, content, repository);
+  if (data.html_url) {
+    cli.ok(`Created: ${data.html_url}`);
+  } else {
+    cli.error(data);
+    process.exit(1);
+  }
+}
+
+export async function pickReport(report, { cli, req }) {
+  const {
+    id, attributes: { title, cve_ids },
+    relationships: { severity, weakness, reporter }
+  } = report;
+  const link = `https://hackerone.com/reports/${id}`;
+  const reportSeverity = {
+    rating: severity?.data?.attributes?.rating || '',
+    cvss_vector_string: severity?.data?.attributes?.cvss_vector_string || '',
+    weakness_id: weakness?.data?.id || ''
+  };
+
+  cli.separator();
+  cli.info(`Report: ${link} - ${title} (${reportSeverity?.rating})`);
+  const include = await cli.prompt(
+    'Would you like to include this report to the next security release?',
+    { defaultAnswer: true });
+  if (!include) {
+    return;
+  }
+
+  const versions = await cli.prompt('Which active release lines this report affects?', {
+    questionType: 'input',
+    defaultAnswer: await getSupportedVersions()
+  });
+
+  let patchAuthors = await cli.prompt(
+    'Add github username of the authors of the patch (split by comma if multiple)', {
```

Comment:
Can we get the PR author instead?

Reply:
What do you mean? Just one username?

Reply:
For now, yes. In a follow-up PR we can get the authors of all commits of a PR and include them as patchAuthors

Reply:
So instead of manually inserting the patch author we ask the PR so we can include the link and the author?

Reply:
Can't we automatically link the report -> pr?

Reply:
I dont think report holds any information about the PR

Reply:
Deal